.coderabbit.yaml

```
@@ -0,0 +1,52 @@
+
+  path_filters:
+    - '!node_modules/**'
+    - '!dist/**'
+    - '!build/**'
+    - '!coverage/**'
+    - '!*.lock'
+
+
+
+  auto_review:
+    enabled: true 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)